llm/llm.go

```diff
@@ -27,6 +30,34 @@ func New(model string, opts api.Options) (LLM, error) {
 		return nil, err
 	}
 
+	switch ggml.FileType {
+	case FileTypeQ5_0, FileTypeQ5_1, FileTypeQ8_0:
+		// Q5_0, Q5_1, and Q8_0 do not support Metal API and will
+		// cause the runner to segmentation fault so disable GPU
+		log.Printf("WARNING: GPU disabled for Q5_0, Q5_1, and Q8_0")
```

Comment:
this is magic – being able to fallback to a "working" runtime instead of a flat out error